### PR TITLE
add schema param to yugabyte integration docs

### PR DESCRIPTION
**Pull request by Ayobami6**
## Description

Please include a summary of the change and the issue it solves. 

**Fixes** #(issue)

## Type of change

(Please delete options that are not relevant)

- [ ] 🐛 Bug fix (non-breaking change which fixes an issue)
- [ ] ⚡ New feature (non-breaking change which adds functionality)
- [ ] 📢 Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] 📄 This change requires a documentation update

### What is the solution?

(Describe at a high level how the feature was implemented)

## Checklist:

- [ ] My code follows the style guidelines(PEP 8) of MindsDB.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation, or created issues to update them.
- [ ] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [ ] I have shared a short loom video or screenshots demonstrating any new functionality.

### CODE REVIEW

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by mindsdbadmin** [comment]
**CLA Assistant Lite bot:** <br/>Thank you for your submission, we really appreciate it. Like many open-source projects, we ask that you sign our [Contributor License Agreement](https://github.com/mindsdb/mindsdb_native/blob/stable/assets/contributions-agreement/individual-contributor.md) before we can accept your contribution. You can sign the CLA by just posting a Pull Request Comment same as the below format.<br/>
   - - -
   I have read the CLA Document and I hereby sign the CLA
   - - -
   <sub>You can retrigger this bot by commenting **recheck** in this Pull Request</sub>

**Comment by Ayobami6** [comment]
I have read the CLA Document and I hereby sign the CLA

Thank you

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by ZoranPandovski** [comment]
recheckcla

**Comment by ZoranPandovski** [comment]
recheckcla

File: docs/data-integrations/yugabytedb.mdx [hunk 1]
```diff
@@ -13,11 +13,12 @@ This handler is implemented using `psycopg2`, a Python library that allows you t
...
-* `password` is the database password.
-* `host` is the host name, IP address, or URL.
-* `port` is the port used to make TCP/IP connection.
-* `database` is the database name.
+- `user` is the database user.
+- `password` is the database password.
+- `host` is the host name, IP address, or URL.
+- `port` is the port used to make TCP/IP connection.
+- `database` is the database name.
+- `schema` Schema to which your table is asscociated.
```
Comment (martyna-mindsdb) [inline docs/data-integrations/yugabytedb.mdx:21]: Please correct this line to the following:

> \- \`schema\` is the schema to which your table belongs.

**Comment by martyna-mindsdb** [comment]
@Ayobami6 
Please implement a small change as mentioned in the review comment above and push it to this PR.

File: docs/data-integrations/yugabytedb.mdx (hunk 1 above)
Comment (Ayobami6) [reply docs/data-integrations/yugabytedb.mdx]: Done

**Pull request merged**